### ISSUE TITLE
Build Battle By @ImagicalGamer

### ISSUE THREAD

**Issue by Taki21** [Inactive-to-Reactive/Team-Notices#1]
Build Battle: by @ImagicalGamer 
U see....
![image](https://cloud.githubusercontent.com/assets/14851895/15917640/8bc12d62-2dcc-11e6-874f-c1320013e853.png)
And we NEED REACTIVITY POWER!

**Comment by Taki21** [comment]
@ImagicalGamer 
@remotevase

**Comment by remotevase** [comment]
True, but IDK how to code buildbattle, so you guys can work on that

**Comment by Ad5001** [comment]
@Taki21 Might help you

**Comment by Taki21** [comment]
lel ya

**Comment by jasonwynn10** [comment]
anyone mind if I join the group?

**Comment by ImagicalGamer** [comment]
tbh I already finished buildbattle but it's private for a one of the networks I work for lmao

**Comment by remotevase** [comment]
@jasonwynn10 sure!

**Comment by kabluinc** [comment]
@ImagicalGamer can u plz share it with me? :)
